test/views/SubmittedReturnsViewSpec.scala

```diff
@@ -631,4 +632,122 @@ class SubmittedReturnsViewSpec extends ViewBaseSpec {
     }
 
   }
+
+  "Rendering the submitted returns page for an agent with the changeClient feature enabled" when {
+
+    mockConfig.features.changeClient(true)
+
+    "multiple years are returned" should {
+
+      val returnYears = Seq(2020, 2019, 2018)
+
+      lazy val exampleReturns: Seq[ReturnObligationsViewModel] =
+        Seq(
+          ReturnObligationsViewModel(
+            LocalDate.parse("2020-01-01"),
+            LocalDate.parse("2020-03-31"),
+            "#001"
+          ),
+          ReturnObligationsViewModel(
+            LocalDate.parse("2020-04-01"),
+            LocalDate.parse("2020-06-30"),
+            "#002"
+          ),
+          ReturnObligationsViewModel(
+            LocalDate.parse("2019-01-01"),
+            LocalDate.parse("2019-03-31"),
+            "#001"
+          ),
+          ReturnObligationsViewModel(
+            LocalDate.parse("2019-04-01"),
+            LocalDate.parse("2019-06-30"),
+            "#002"
+          ),
+          ReturnObligationsViewModel(
+            LocalDate.parse("2018-01-01"),
+            LocalDate.parse("2018-03-31"),
+            "#001"
+          ),
+          ReturnObligationsViewModel(
+            LocalDate.parse("2018-04-01"),
+            LocalDate.parse("2018-06-30"),
+            "#002"
+          )
+        )
+
+      lazy val view = views.html.returns.submittedReturns(
+        VatReturnsViewModel(returnYears, 2020, exampleReturns, hasNonMtdVatEnrolment = true, "999999999")
+      )(fakeRequestWithClientsVRN, messages, mockConfig, Lang("en-GB"), agentUser)
+
+      lazy implicit val document: Document = Jsoup.parse(view.body)
+
+      "have a tab for 2020" should {
+
+        "have the text '2020'" in {
+          elementText(Selectors.tabOne) should include("2020")
+        }
+
+        "contain visually hidden text" in {
+          elementText(Selectors.tabOneHiddenText) shouldBe "Currently viewing returns from 2020"
+        }
+
+        "display a 'change client' link" in {
+          document.getElementById("changeClient").text() shouldBe "Change client"
+          document.getElementById("changeClient").attr("href") shouldBe mockConfig.agentClientLookupUrl(mockConfig.agentClientActionUrl)
+        }
+
+        "display a 'finish' button" in {
+          document.getElementById("finish").text() shouldBe "Finish"
+          document.getElementById("finish").attr("href") shouldBe mockConfig.agentClientActionUrl
+        }
+
+      }
+
+      "have a tab for 2019" should {
```

Comment:
No need to retest all this stuff as its done in existing tests - just have checks for dynamic bits you've added